pubspec.yaml

```
@@ -1,11 +1,11 @@
 name: outline_material_icons
 description: Provides the new outline theme of material design icons from Google.
-version: 0.1.1
+version: 0.2.0-nullsafety
```

Null safety is now stable so we can remove the prerelease

```suggestion
version: 0.2.0
```

---

And I was questening myself why others do this "-nullsafety" but now it's clear for me xD thx!